stages/test/test_containers_storage_conf.py

```diff
@@ -119,4 +121,5 @@ def test_schema_validation_containers_storage_conf(test_data, storage_test_data,
     else:
         assert res.valid is False
         err_msgs = [e.as_dict()["message"] for e in res.errors]
-        assert expected_err in err_msgs[0] or expected_err in err_msgs[1]
+        assert any(re.search(expected_err, err_msg)
```

Comment:
It's a tiny bit iffy to do this if any of the current expected messages contain any accidental regular expression expressions. If you have checked that they don't then that's OK.

Reply:
Right I've just verified this as well (not that many parameters here) so this is OK.

Reply:
I did check but you make a good point - I wonder if we can find something that makes this more robust so that if someone later adds a special char accidentially we catch this in the right way